model/page.go

```diff
@@ -31,6 +31,17 @@ type Page struct {
 	BackTo                           BackTo           `json:"back_to"`
 	SearchNoIndexEnabled             bool             `json:"search_no_index_enabled"`
 	NavigationContent                []NavigationItem `json:"navigation_content"`
+	ReleasePageData                  ReleasePage      `json:"release_page"`
+}
+
+// ReleasePage contains all information needed to render gmt dataLayer for release-page
+type ReleasePage struct {
```

Comment:
This is very specific but your properties are not. If there's value in recording these types then a more generic approach would be preferable.